pkg/tests/end_to_end_tests/golden_files_test.go

```diff
@@ -17,9 +17,10 @@ import (
 )
 
 var outputDifferWithoutTimescale = map[string]bool{"info_view": true}
-var outputDifferWithMultinode = map[string]bool{"views": true, "info_view": true, "support": true}
+var outputDifferWithMultinode = map[string]bool{"views": true, "info_view": true}
 var outputDifferWithExtension = map[string]bool{"support": true}
 var requiresTimescaleDB = map[string]bool{"views": true, "info_view": true, "support": true}
+var requiresSingleNode = map[string]bool{"support": true}
```

Comment:
Can we please group them into a single var declaration?

Reply:
No, I don't like single var declarations in general